Source/Pages/Gallery/BottomPager/YPMenuItem.swift

```diff
@@ -42,7 +42,7 @@ final class YPMenuItem: UIView {
         
         textLabel.style { l in
             l.textAlignment = .center
-            l.font = UIFont.systemFont(ofSize: 17, weight: UIFont.Weight.medium)
+            l.font = UIFont.systemFont(ofSize: 17, weight: .semibold)
```

Comment:
What is this for?

Reply:
I guess this is to match Instagram closer. Looks to me that instagram font is a bit bolder than ours.